*** src/sample/pgpool.conf.sample	2022-11-06 16:39:24.710083883 +0900
--- pgpool.conf.sample	2022-11-06 16:37:43.030393436 +0900
***************
*** 40,46 ****
  #port = 9999
                                     # Port number
                                     # (change requires restart)
! #unix_socket_directories = '/tmp'
                                     # Unix domain socket path(s)
                                     # The Debian package defaults to
                                     # /var/run/postgresql
--- 40,46 ----
  #port = 9999
                                     # Port number
                                     # (change requires restart)
! unix_socket_directories = '/var/run/postgresql'
                                     # Unix domain socket path(s)
                                     # The Debian package defaults to
                                     # /var/run/postgresql
***************
*** 72,78 ****
  #pcp_port = 9898
                                     # Port number for pcp
                                     # (change requires restart)
! #pcp_socket_dir = '/tmp'
                                     # Unix domain socket path for pcp
                                     # The Debian package defaults to
                                     # /var/run/postgresql
--- 72,78 ----
  #pcp_port = 9898
                                     # Port number for pcp
                                     # (change requires restart)
! pcp_socket_dir = '/var/run/postgresql'
                                     # Unix domain socket path for pcp
                                     # The Debian package defaults to
                                     # /var/run/postgresql
***************
*** 696,702 ****
                                      # Authentication key for watchdog communication
                                      # (change requires restart)
  
! #wd_ipc_socket_dir = '/tmp'
                                      # Unix domain socket path for watchdog IPC socket
                                      # The Debian package defaults to
                                      # /var/run/postgresql
--- 696,702 ----
                                      # Authentication key for watchdog communication
                                      # (change requires restart)
  
! wd_ipc_socket_dir = '/var/run/postgresql'
                                      # Unix domain socket path for watchdog IPC socket
                                      # The Debian package defaults to
                                      # /var/run/postgresql
